lib/cosmos/system/system.rb

```diff
@@ -612,13 +608,57 @@ def load_packets(configuration_name = nil)
         else
           @config.name = md5_string
         end
+
         Cosmos.marshal_dump(marshal_filename, @config)
+        setup_system_meta()
       end
 
       @initial_config = @config unless @initial_config
       save_configuration()
     end
 
+    def setup_system_meta
+      # Ensure SYSTEM META is defined and defined correctly
+      begin
+        tlm_meta = @telemetry.packet('SYSTEM', 'META')
+        cmd_meta = @commands.packet('SYSTEM', 'META')
+        item = tlm_meta.get_item('PKTID')
+        raise "PKTID Incorrect" unless item.bit_size == 8 and item.bit_offset == 0
+        item = cmd_meta.get_item('PKTID')
+        raise "PKTID Incorrect" unless item.bit_size == 8 and item.bit_offset == 0
+        item = tlm_meta.get_item('CONFIG')
+        raise "CONFIG Incorrect" unless item.bit_size == 256 and item.bit_offset == 8
+        item = cmd_meta.get_item('CONFIG')
+        raise "CONFIG Incorrect" unless item.bit_size == 256 and item.bit_offset == 8
+      rescue
+        Logger.error "SYSTEM META not defined or defined incorrectly - defaulting"
```

Comment:
This begs the question whether it needs to be defined at all if we're just going to define it ourselves if it's wrong.